aero.minova.core.application.system.service/src/test/java/aero/minova/core/application/system/controller/SetupServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package aero.minova.core.application.system.controller;
+
+import java.io.File;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.util.List;
+import java.util.Scanner;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import com.mysql.cj.jdbc.CallableStatement;
+
+import aero.minova.core.application.system.service.FilesService;
+import aero.minova.core.application.system.service.SetupService;
+import aero.minova.core.application.system.sql.SystemDatabase;
+
+public class SetupServiceTest {
+
+	@Mock
+	private SystemDatabase database;
+	@Mock
+	private Connection connection;
+	@Mock
+	private FilesService service;
+
+	@Test
+	public void testExampleProject() {
+		InputStream inputStream = getClass().getClassLoader().getResourceAsStream("exampleDependencyList.txt");
+		String text = null;
+		try (Scanner scanner = new Scanner(inputStream, StandardCharsets.UTF_8.name())) {
+			text = scanner.useDelimiter("\\A").next();
+		}
+		new SetupService().parseDependencyList(text);
+	}
+
+	@Test
+	public void testReadSetups() throws Exception {
+		MockitoAnnotations.initMocks(this);
+
+		// Die Setup-Dateien liegen im setup-Ordner.
+		String url = getClass().getClassLoader().getResource("setup").toURI().getPath();
+		Path setupPath = new File(url.substring(0, url.lastIndexOf("/"))).toPath();
+		Mockito.when(service.getSystemFolder()).thenReturn(setupPath);
+		Mockito.when(database.getConnection()).thenReturn(connection);
+		Mockito.when(connection.prepareCall(Mockito.any())).thenReturn(Mockito.mock(CallableStatement.class));
+
+		SetupService testSubject = new SetupService();
+		testSubject.service = service;
+		testSubject.database = database;
+		InputStream inputStream = getClass().getClassLoader().getResourceAsStream("exampleDependencyList.txt");
+		String text = null;
+		try (Scanner scanner = new Scanner(inputStream, StandardCharsets.UTF_8.name())) {
+			text = scanner.useDelimiter("\\A").next();
```

Comment:
Wofür wird `useDelimiter("\\A")` verwendet?
Laut Javadoc ist `\A` `The beginning of the input`.

Reply:
Das hab ich von dir so übernommen, aber ich kann es auch ändern.

Reply:
Ah jetzt erinnere ich mich.
Der Scanner benötigt auf jeden Fall einen Delimiter und wir müssen den Scanner verwenden.